src/transport_ofi.c

```diff
@@ -1500,6 +1488,83 @@ int shmem_transport_init(void)
 int shmem_transport_startup(void)
 {
     int ret;
+    int i;
+    int  num_on_node = 0;
+
+    if (shmem_internal_params.OFI_STX_AUTO) {
+
+        num_on_node = shmem_node_util_count_local_pes();
+        if (num_on_node <= 0) {
+            RAISE_ERROR_STR("Failed to find any node-local PEs");
+            return 1;
```

Comment:
Looks like another instance of the "return 1" not actually happening?

Reply:
Yeah, either we should return on error or abort.  Currently the function signature indicates it will return, but instead aborts.